workflows/exomeseq.cwl

```diff
@@ -96,12 +96,40 @@ outputs:
     outputSource: variant_discovery/variant_recalibration_snps_indels_vcf
     doc: "The output filtered and recalibrated VCF file in which each variant is annotated with its VQSLOD value"
 steps:
+  make_target_interval_list:
+    run: ../tools/picard-BedToIntervalList.cwl
+    requirements:
+      - class: ResourceRequirement
+        coresMin: 1
+        ramMin: 4000
+        outdirMin: 12000
+        tmpdirMin: 12000
+    in:
+      input_file: intervals
+      reference_sequence: reference_genome
+    out:
+      - output_interval_list_file
+  make_bait_interval_list:
+    run: ../tools/picard-BedToIntervalList.cwl
+    requirements:
+      - class: ResourceRequirement
+        coresMin: 1
+        ramMin: 4000
+        outdirMin: 12000
+        tmpdirMin: 12000
+    in:
+      input_file: primary_intervals
+      reference_sequence: reference_genome
+    out:
+      - output_interval_list_file
```

Comment:
Can these tool steps be abstracted into a workflow like `exomeseq-00-prepare-reference-data` for consistency with the rest of the steps?